### PR TITLE
meta: Bump version to 0.5 'Taygete'

### PR DESCRIPTION
**Pull request by Dennisbonke**
Starting with the 0.5 'Taygete' development cycle. Also doing some cleanup tasks.

### CODE REVIEW

File: Assets/Scenes/TutorialLevel.unity [hunk 1]
```diff
@@ -10447,84 +10446,6 @@ PrefabInstance:
       objectReference: {fileID: 0}
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: fe9a12e8b29f2cb5ba644d961fd951ce, type: 3}
---- !u!1001 &1123240534
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:10463]: Is this the 2nd Adam that is removed?

File: Assets/Scenes/TutorialLevel.unity [hunk 2]
```diff
@@ -9464,7 +9464,6 @@ Transform:
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children:
   - {fileID: 818750476}
-  - {fileID: 1123240535}
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:9480]: What is this file that is deleted?

File: Assets/Scenes/TutorialLevel.unity [hunk 3]
```diff
@@ -6114,7 +6114,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
+  m_IsActive: 1
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity]: What is activated here?

File: Assets/Scenes/MainMenu.unity [hunk 4]
```diff
@@ -3396,7 +3396,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
+  m_IsActive: 1
```
Comment (Westlanderz) [inline Assets/Scenes/MainMenu.unity:3399]: Only wondering what is activated?

File: Assets/Scenes/MainMenu.unity [hunk 5]
```diff
@@ -3406,7 +3406,7 @@ MonoBehaviour:
   m_GameObject: {fileID: 1935625980}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: 0025fdbd4724a23b7bd493093a7e4968, type: 3}
+  m_Script: {fileID: 11500000, guid: b7725820b005fc2afbc3870fef3a04d8, type: 3}
```
Comment (Westlanderz) [inline Assets/Scenes/MainMenu.unity:3409]: Why is this changed?

File: Assets/Scenes/TutorialLevel.unity [hunk 6]
```diff
@@ -16678,17 +16599,6 @@ Transform:
   m_CorrespondingSourceObject: {fileID: 7930691839557344146, guid: fe9a12e8b29f2cb5ba644d961fd951ce, type: 3}
   m_PrefabInstance: {fileID: 1621152295}
   m_PrefabAsset: {fileID: 0}
---- !u!114 &1820880696 stripped
-MonoBehaviour:
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:16710]: What is this deletion?

File: Assets/Scenes/TutorialLevel.unity [hunk 7]
```diff
@@ -15819,7 +15732,7 @@ MonoBehaviour:
   cutsceneController_: {fileID: 984060224}
   unitGridCombatArray_:
   - {fileID: 2046354194}
-  - {fileID: 1820880696}
+  - {fileID: 0}
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:15735]: `fileID: 0` ??

**Review by Westlanderz — COMMENTED** [review]
Just some questions.

File: Assets/Scenes/MainMenu.unity (hunk 5 above)
Comment (Dennisbonke) [reply Assets/Scenes/MainMenu.unity:3409]: Script probably got moved, is for the sound controller.

File: Assets/Scenes/MainMenu.unity (hunk 4 above)
Comment (Dennisbonke) [reply Assets/Scenes/MainMenu.unity:3399]: The main menu sound controller.

File: Assets/Scenes/TutorialLevel.unity (hunk 7 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:15735]: Good catch, this was the second Adam, I'll fix this.

File: Assets/Scenes/TutorialLevel.unity (hunk 6 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:16710]: The second Adam being deleted as discussed.

File: Assets/Scenes/TutorialLevel.unity [hunk 8]
```diff
@@ -8655,7 +8650,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.032, y: 0.24354449}
   m_AnchorMax: {x: 0.18400002, y: 0.47155875}
-  m_AnchoredPosition: {x: 0.000030517578, y: 0}
+  m_AnchoredPosition: {x: 0, y: 0}
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:8653]: Not sure what is moved here, but I assume that this leaves every object in the correct location if this is a parent object that is moved.

File: Assets/Scenes/TutorialLevel.unity [hunk 9]
```diff
@@ -4259,7 +4259,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.81600004, y: 0.026612679}
   m_AnchorMax: {x: 0.97700006, y: 0.1902057}
-  m_AnchoredPosition: {x: -2.082489, y: -0.35250854}
+  m_AnchoredPosition: {x: -2.0824585, y: -0.35250854}
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:4262]: Same move comment here.

File: Assets/Scenes/TutorialLevel.unity [hunk 10]
```diff
@@ -4452,7 +4452,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.8223292, y: 0.20500001}
   m_AnchorMax: {x: 0.9595612, y: 0.23041128}
-  m_AnchoredPosition: {x: -0.000030517578, y: 0.39248657}
+  m_AnchoredPosition: {x: 0, y: 0.39248657}
```
Comment (Westlanderz) [inline Assets/Scenes/TutorialLevel.unity:4455]: Same move comment here.

**Review by Westlanderz — COMMENTED** [review]
Some new questions I had not spot in the first review.

File: Assets/Scenes/TutorialLevel.unity (hunk 8 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:8653]: Something in the cutscene, `LeftChar` to be precise.

File: Assets/Scenes/TutorialLevel.unity (hunk 9 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:4262]: Again in the cutscene, `ButtonEnd` this time.

File: Assets/Scenes/TutorialLevel.unity (hunk 10 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:4455]: And also in the cutscene, `RightName` this time.

File: Assets/Scenes/TutorialLevel.unity (hunk 1 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:10463]: I'd guess yes.

File: Assets/Scenes/TutorialLevel.unity (hunk 2 above)
Comment (Dennisbonke) [reply Assets/Scenes/TutorialLevel.unity:9480]: Also the player I presume.

**Review by Westlanderz — APPROVED** [review]
LGTM
